kernel/layout/set.php

```diff
@@ -17,7 +17,7 @@
 
 $Result = array();
 $Result['content'] = '';
-$Result['rerun_uri'] = '/' . implode( '/', array_splice( $Params['Parameters'], 1 ) ) . $userParamString;
+$rerunURL = '/' . implode( '/', array_splice( $Params['Parameters'], 1 ) ) . $userParamString;
```

Comment:
what is the point of this change?

Reply:
Prevent BC break: only return "rerun_uri" in the module result if the layout exists, since the PR makes it so the kernel only checks "rerun_uri" rather than the module's exit status. Alternatively we could just move this code further down to where $rerunURL is used.

Reply:
> Alternatively we could just move this code further down to where $rerunURL is used.

👍 that would be cleaner / easier to read.

Reply:
Done, thanks!